modules/gdnative/nativescript/nativescript.cpp

```diff
@@ -1599,18 +1599,20 @@ bool NativeScriptLanguage::handles_global_class_type(const String &p_type) const
 }
 
 String NativeScriptLanguage::get_global_class_name(const String &p_path, String *r_base_type, String *r_icon_path) const {
-	Ref<NativeScript> script = ResourceLoader::load(p_path, "NativeScript");
-	if (script.is_valid()) {
+	if (!String(p_path).empty()) {
```

Comment:
does `!p_path.empty()` not work here?

Reply:
And you can avoid indenting everything by doing instead:
```cpp
if (p_path.empty()) {
	String();
}
```

Reply:
No nevermind, bad idea :)

Reply:
you guys are fast 😄 , done.